### PR TITLE
run code in Docker (WP-686)

### PR DESCRIPTION
**Pull request by vsolovei-smartling**
extract release.sh to prevent escaping errors

### CODE REVIEW

**Review by PavelLoparev — APPROVED** [review]

File: Buildplan/Dockerfile-Jenkins [hunk 1]
```diff
@@ -67,8 +67,10 @@ ENV COMPOSER_H_DIR="${HOME}/.composer"
 ENV PLUGIN_DIR="${WP_PLUGINS_DIR}/smartling-connector"
 ENV LOCAL_GIT_DIR="/plugin-dir"
 
-COPY Buildplan/build.sh /build.sh
-RUN chmod +x /build.sh
+COPY Buildplan/release.sh /release.sh
+COPY Buildplan/test.sh /test.sh
+RUN chmod +x /release.sh
+RUN chmod +x /test.sh
```
Comment (dimitrystd) [inline Buildplan/Dockerfile-Jenkins:73]: JFYI, can be single line
```suggestion
RUN chmod +x /release.sh /test.sh
```

File: Buildplan/release.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+cp -r ../inc/config ./inc
+cp -r ../inc/Smartling ./inc
+cp -r ../js .
+cp -r ../languages .
+
+svn add --force * --auto-props --parents --depth infinity -q
+TAG=`grep '* Version' smartling-connector.php | sed 's/ \\* Version: *//'`
+svn status
+echo "About to commit $TAG with these changes in a minute!"
+sleep 1m
```
Comment (dimitrystd) [inline Buildplan/release.sh:15]: Didn't get this magic. Is it the last chance to interrupt jenkins buid manually? If so, then I would recommend:
1. Checkout svn in some special folder inside container, but map this folder to jenkins local drive. Like
    ```
    docker run ... -v $PWD/plugin-from-svn:/svn/plugin ...
    ```
2. Remove commit steps from this script
3. Ask question in jenkins. If user confirmed then run docker again with commit command
    ```
    docker run ... -v $PWD/plugin-from-svn:/svn/plugin ... wordpress-localization-plugin-php74:latest /commit_and_tag.sh
    ```
4. Jenkins post-deploy steps: delete `$PWD/plugin-from-svn` (if jenkins doesn't do this)

Comment (PavelLoparev) [reply Buildplan/release.sh]: I completely overlooked this(

Comment (vsolovei-smartling) [reply Buildplan/release.sh]: Done

**Pull request merged**